### ISSUE TITLE
libthcli.so

### ISSUE THREAD

**Issue by sgamel** [ifxdb/node-ifx_db#6]
I get an error when trying to load Ifx_db - Any ideas why?  I have the sdk installed, and the file is there.

/app/mqmon/node_modules/ifx_db/node_modules/bindings/bindings.js:79
        throw e
        ^

Error: libthcli.so: cannot open shared object file: No such file or directory
    at Error (native)
    at Object.Module._extensions..node (module.js:460:18)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at bindings (/app/mqmon/node_modules/ifx_db/node_modules/bindings/bindings.js:74:15)
    at Object.<anonymous> (/app/mqmon/node_modules/ifx_db/lib/odbc.js:29:31)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)

**Comment by msatyan** [comment]
The driver is built with NodeJS version 4x. 
I believe you may be using higher NodeJS version. 

The v4x driver is not compatible with higher NodeJS versions
Soon we will be working on upgrading the driver version to Node v6 and v7

**Comment by sgamel** [comment]
Thanks for response.  This ended up being an environmental issue.  I did have other issues getting this to build, because of npm version issues.
If someone runs in to the same issue trying to build with node v4.3.0 - try updating npm, reinstall node-gyp and try the build again.  It will warn you to update your node version, but it is fine.
So, if you get a path error on this step --> node-gyp configure -v
I ran this -> npm update npm -g
then this -> npm install node-gyp -g
delete the ./build and try again.  
I was building this for a server that I did not have the option of upgrading node/npm/gcc, so I had to do it this way to get it to work.  I then just zipped up my node_modules and dropped it on the other box, along with the CSDK and I was working after that.
